projects/ppwcode/ng-common-components/src/lib/search-filter/search-filter.component.ts

```diff
@@ -19,5 +19,14 @@ export class SearchFilterComponent {
 
     // Outputs
     public search: OutputEmitterRef<void> = output<void>()
+    /**
+     * @deprecated This output will be removed in v20. It should be replaced with the `restart` output.
+     */
     public reset: OutputEmitterRef<void> = output<void>()
+    public restart: OutputEmitterRef<void> = output<void>()
+
+    public resetClicked(): void {
```

Comment:
This method can be avoided by assigning the `restart` output to the deprecated `reset` output:

```typescript
public restart: OutputEmitterRef<void> = output<void>()
/**
 * @deprecated This output will be removed in v20. It should be replaced with the `restart` output.
 */
public reset: OutputEmitterRef<void> = this.restart;
```

In the template, you can then just emit on `restart` and that will also emit `reset`: `restart.emit()` as they are working on the same instance.